metadata.yaml

```diff
@@ -24,6 +24,11 @@ issues: https://github.com/canonical/self-signed-certificates-operator/issues
 provides:
   certificates:
     interface: tls-certificates
+  send-ca-cert:
+    interface: mutual_tls
```

Comment:
This can be changed to `certificate-exchange` now. The renamed library is published.

Reply:
Can't find it. Could you please point me to where it's at? Tried:
```
charms.mutual_tls_interface.v0.certificate_exchange
charms.mutual_tls_interface.v1.certificate_exchange
charms.mutual_tls_interface.v2.certificate_exchange
charms.certificate_exchange__interface.v0.certificate_exchange
charms.certificate_exchange__interface.v1.certificate_exchange
```

Reply:
Let's make sure we all agree on the interface name before going too far in on implementation ([ref](https://github.com/canonical/charm-relation-interfaces/pull/95))